packages/shell-chrome/src/backend.js

```diff
@@ -125,8 +134,8 @@ function discoverComponents(isThroughMutation = false) {
             rootEl.__alpineDevtool = {}
         }
 
-        if (!isThroughMutation) {
-            rootEl.__alpineDevtool.id = Math.floor(Math.random() * 100000 + 1)
+        if (!rootEl.__alpineDevtool.id) {
+            rootEl.__alpineDevtool.id = parseInt(Date.now() + Math.floor(Math.random() * 1000000), 10)
```

Comment:
can we move this to a util function? I'm also tempted to use a generic UUID/short UUID solution instead of trying to create our own custom id generator

Reply:
Just saw this comment actually. I can extract it. Just something like `generateRandomNumber()` maybe?

Reply:
`createComponentId` would work as well